conf/pacific/cephadm/tier-1_5node_cephadm_bootstrap.yaml

```diff
@@ -39,5 +39,10 @@ globals:
         no-of-volumes: 6
         disk-size: 15
       node4:
+        role:
+          - rgw
+        no-of-volumes: 6
+        disk-size: 15
```

Comment:
Any particular reason why `disks` are being added to the VM not having `osd` role

Reply:
No particular reason. I was just testing out, I have removed the disks